components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,145 @@ toc = true
     weight = 30
 +++
 
-This guide shows you how to manage Chef Automate IAM v2 policies and roles from the command line.
+This API reference details Chef Automate IAM v2 features from the command line.
+If you are not already on IAM v2, see the [IAM v2 User's Guide]({{< relref "iam-v2-guide.md#upgrade-to-iam-v2" >}}) to upgrade.
 Any command with a browser equivalent includes brief directions for locating it on the corresponding page.
 See the [IAM v2 Overview]({{< relref "iam-v2-overview.md" >}}) for an overview and exposition of new concepts.
 See the [IAM v2 User's Guide]({{< relref "iam-v2-guide.md" >}}) for more information on setting up and using IAM v2 features.
 
-## Prerequisites
-
-1. Upgrade to IAM v2 using the [IAM v2 User's Guide]({{< relref "iam-v2-guide.md#upgrade-to-iam-v2" >}}).
-
-2. _Optional_ Install the [jq](https://stedolan.github.io/jq/) JSON processor to ensure readable output.
-
-3. Generate a new admin token for IAM v2.
-   (The v1 command `chef-automate admin-token` will not work for v2, though any existing v1 admin tokens will still work.) Use:
+<!--The TOC below is auto-generated by the VSCode "Markdown TOC" plugin; do not edit manually.
+With the plugin installed, this TOC will update automatically when you save the file! -->
+<!-- TOC depthFrom:2 depthTo:3 -->
```

Comment:
Per this comment, I have included an auto-generated Table-of-Contents. I am not aware of this is good practice in a Hugo document, so if Hugo has an alternate way of doing this, great. But it is vital in this file to help folks quickly find what they want.